packages/repository/src/relations/has-many/has-many-through.repository.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Count, DataObject, Entity, Filter, Options, Where} from '../..';
+
+/**
+ * CRUD operations for a target repository of a HasManyThrough relation
+ */
+export interface HasManyThroughRepository<
+  Target extends Entity,
+  TargetID,
+  Through extends Entity
+> {
+  /**
+   * Create a target model instance
+   * @param targetModelData - The target model data
+   * @param options - Options for the operation
+   * @returns A promise which resolves to the newly created target model instance
+   */
+  create(
+    targetModelData: DataObject<Target>,
+    options: Options & {
```

Comment:
```suggestion
    options?: Options & {
```
After some experiments with this API, i think options should be optional here too.